src/installer/CMakeLists.txt

```diff
@@ -44,7 +45,8 @@ include(InstallRequiredSystemLibraries)
 set(CMAKE_INSTALL_PREFIX ${INSTALL_ROOT})
 
 # Add elasticodbc project to installer
-install(TARGETS elasticodbc DESTINATION bin)
+install(TARGETS elasticodbc DESTINATION bin COMPONENT "Driver")
+install(FILES "${PROJECT_ROOT}/bin64/dsn_installer" DESTINATION bin COMPONENT "Driver")
```

Comment:
Add check for OS

Reply:
Yep, seems like this is why the build failed. I'm probably going to combine OS-specific sections as much as possible